packages/wdio-cucumber-framework/src/hookRunner.js

```diff
@@ -0,0 +1,41 @@
+import { executeHooksWithArgs } from '@wdio/config'
+import CucumberEventListener from './cucumberEventListener'
+
+export default class HookRunner {
+    gherkinDocEvents = []
+
+    constructor (eventBroadcaster, config) {
+        this.config = config
+        this.eventListener = new CucumberEventListener(eventBroadcaster)
+            .on('before-feature', this.handleBeforeFeature.bind(this))
+            .on('before-scenario', this.handleBeforeScenario.bind(this))
+            .on('before-step', this.handleBeforeStep.bind(this))
+            .on('after-step', this.handleAfterStep.bind(this))
+            .on('after-scenario', this.handleAfterScenario.bind(this))
+            .on('after-feature', this.handleAfterFeature.bind(this))
+    }
+
+    handleBeforeFeature (uri, feature) {
```

Comment:
we have lots of parameters for every hook here, like uri, feature, scenario, etc, however we always passing the last param instead of all of them.
Should it be changes to something like `return executeHooksWithArgs(this.config.beforeFeature, [uri, feature])` or I'm missing something?